packages/pill/index.js

```diff
@@ -0,0 +1,104 @@
+import "@warp-ds/icons/elements/close-16";
+import { activateI18n } from "../i18n";
+import { classNames } from "@chbphone55/classnames";
+import { html } from "lit";
+import { i18n } from "@lingui/core";
+import { messages as enMessages } from "./locales/en/messages.mjs";
+import { messages as fiMessages } from "./locales/fi/messages.mjs";
+import { messages as nbMessages } from "./locales/nb/messages.mjs";
+import { pill as ccPill } from "@warp-ds/css/component-classes";
+import WarpElement from "@warp-ds/elements-core";
+
+class WarpPill extends WarpElement {
+  static styles = [WarpElement.styles];
+
+  static properties = {
+    canClose: { type: Boolean },
+    suggestion: { type: Boolean },
+    openSRLabel: { type: String },
+    closeSRLabel: { type: String },
```

Comment:
These properties would be bound to lowercase attributes like `canclose` or `opensrlabel`. Whenever we'd have multi-word properties, we would bind camelCase properties with kebab-case attributes using a `kebabCaseAttributes` wrapper on `WarpElement`. If we use that approach here for the sake of consistency, then maybe `openSRLabel` could be renamed so you don't need to pass an attribute like `open-s-r-label="..."`? Wdyt?

Reply:
Yeah, that sounds good.

Reply:
What do you think about `closeSrLabel`/`openSrLabel` so we can have `close-sr-label`/`open-sr-label` attributes?